helpers.php

```diff
@@ -548,6 +657,20 @@ function updateTransactionIdForParentSubscription($id_order, $paymentId)
     Db::getInstance()->Execute($sql);
 }
 
+/**
+ * @param $id_order
+ * @param $paymentId
+ *
+ * @return void
+ */
+function updateParentTransIdChildOrder($id_order, $paymentId)
+{
+    $sql = 'UPDATE 
+    ' . _DB_PREFIX_ . 'altapay_child_order SET payment_id = \'' . pSQL($paymentId) . '\' WHERE id_order='
+        . (int) $id_order . ' LIMIT 1';
```

Comment:
Same comments as above.

Reply:
We have added a validation for non-zero positive integer ✅

Reply:
We are still casting

![image](https://github.com/AltaPay/plugin-prestashop/assets/1629945/aa920a56-fb52-4c34-be0e-28de0a586df8)

Reply:
Done